components/observations/ObservationsFilterForm.tsx

```diff
@@ -323,12 +323,13 @@ export const ObservationsFilterForm: React.FunctionComponent<ObservationsFilterF
                     </View>
                   )}
                   {mapLayer && (
+                    // TODO: Render in two groups with respective labels: <Center_ID> Forecast Zones & Other Regions
```

Comment:
@rustynwac let's pin down the UX we want for the two sets of zones with an eye to what the websites are doing.

Reply:
@rustynwac @stevekuznetsov did we decide how we want to represent the zones?